### DISCUSSION TITLE
Players can just place lava on top of players and kill them

### DISCUSSION THREAD

**Discussion by iiMustafaaa** [GriefPrevention/GriefPrevention#1609] [Q&A]
I have a server where there is a pvp toggle option. Players can turn it off and on 

but some players just use lava and kill ppl, this is a big issue and I need a fix. So please let me know how to disable this. Where I can completely disable lava or If I could make lava a inside claim only thing

let me know 

thanks :)

**Comment by bigpresh** [comment]
Look in your config for the `GriefPrevention.PvP.AllowLavaDumpingNearOtherPlayers` setting.

The default values should prohibit it in PvP-enabled worlds, and allow it elsewhere:

```
[davidp@headshrinker:GriefPreventionData]$ yq .GriefPrevention.PvP.AllowLavaDumpingNearOtherPlayers config.yml
{
  "PvPWorlds": true,
  "NonPvPWorlds": false
}
```

Change to suit your requirements.

**Reply by iiMustafaaa** [reply]
But I want to disable lava dumping near players all together would this make it so, players can’t lava dump other players but still be able to use lava?

**Reply by bigpresh** [reply]
If you set both `PvPWorlds` and `NonPvPWorlds` to `true`, then yes, it should prevent dumping lava near other players even in non-PvP worlds (where it typically wouldn't really matter...).

**Reply by iiMustafaaa** [reply]
Umm that’s the issue

I have tried putting it to false and true but no matter what they are able to dump lava on other ppl, this is major issue for now firetick is off 

But could you help me with this 

I will list the plug-ins we have on the server that might conflict and allow lava dump
![Uploading C849A559-E35C-4A3D-BB02-1B2EF51FD46E.jpeg…]()
